### ISSUE TITLE
General Errors: htmlParseEntityRef: no name  need help

### ISSUE THREAD

**Issue by sultansaga** [Automattic/facebook-instant-articles-wp#843]
<!doctype html><html dir="rtl">
  <head>
    <link rel="canonical" href="https://ehsaas.eu/2017/11/30/%d9%be%d8%aa%d8%b1-%da%be%d9%86-%d8%a7%db%92-%da%a9%da%be%da%91%da%a9%db%8c-%da%a9%d8%af%db%92-%d9%86%db%8c%d8%a6%da%ba-%da%a9%da%be%d9%84%d9%86%db%8c/"/>
    <meta charset="utf-8"/>
    <meta property="op:generator" content="facebook-instant-articles-sdk-php"/>
    <meta property="op:generator:version" content="1.6.2"/>
    <meta property="op:generator:application" content="facebook-instant-articles-wp"/>
    <meta property="op:generator:application:version" content="4.0.6"/>
    <meta property="op:generator:transformer" content="facebook-instant-articles-sdk-php"/>
    <meta property="op:generator:transformer:version" content="1.6.2"/>
    <meta property="op:markup_version" content="v1.0"/>
    <meta property="fb:use_automatic_ad_placement" content="enable=true ad_density=default"/>
    <meta property="fb:article_style" content="Ehsaas"/>
  <style type="text/css">
img.wp-smiley,
img.emoji {
	display: inline !important;
	border: none !important;
	box-shadow: none !important;
	height: 1em !important;
	width: 1em !important;
	margin: 0 .07em !important;
	vertical-align: -0.1em !important;
	background: none !important;
	padding: 0 !important;
}
</style>

<link rel='stylesheet' id='jetpack_css-css'  href='https://ehsaas.eu/wp-content/plugins/jetpack/css/jetpack.css?ver=5.6' type='text/css' media='all' />
</head>
  <body>
    <article>
      <header>
        <figure>
          <img src="https://i0.wp.com/ehsaas.eu/wp-content/uploads/2017/11/IMG_0699.jpg?fit=1536%2C1013&amp;ssl=1"/>
        </figure>
        <h1>پتر ھن اے کھڑکی کدے نیئں کھلنی..!!</h1>
        <time class="op-published" datetime="2017-11-30T10:40:51+00:00">November 30th, 10:40am</time>
        <time class="op-modified" datetime="2017-11-30T10:40:51+00:00">November 30th, 10:40am</time>
        <address><a href="http://ehsaas.eu">ijaz amaghrabi</a></address>
        <h3 class="op-kicker">Articles</h3>
        <figure class="op-ad">
          <iframe src="https://www.facebook.com/adnw_request?placement=709812199223510_713256325545764&amp;adtype=banner300x250" width="300" height="250"></iframe>
        </figure>
      </header>
      <p>بہت عرصہ مجھے اس راز کا پتہ ھی نہ چل سکا کہ ماں کو میرے آنے کی خبر کیسے ھو جاتی ھے..<br/>میں سکول سے آتا تو دستک دینے سے ہہلے دروازہ کھل جاتا.. کالج سے آتا تو دروازے کے قریب پہنچتے ھی ماں کا خوشی سے دمکتا چہرہ نظر آ جاتا.. وہ پیار بھری مسکراہٹ سے میرا استقبال کرتی.. دعائیں دیتی.. اور پھر میں صحن میں اینٹوں سے بنے چولہے کے قریب بیٹھ جاتا.. ماں گرما گرم روٹی بناتی اور میں مزے سے کھاتا.. جب میرا پسندیدہ کھانا پکا ھوتا تو ماں کہتی ” چلو مقابلہ کریں “<p>یہ مقابلہ بہت دلچسپ ھوتا تھا..<p>ماں روٹی چنگیر میں رکھتی اور کہتی ” اب دیکھتے ھیں کہ پہلے میں دوسری روٹی پکاتی ھوں یا تم اسے ختم کرتے ھو.. ” ماں کچھ اس طرح روٹی پکاتی ‘ ادھر آخری نوالہ میرے منہ میں جاتا اور ادھر تازہ تازہ اور گرما گرم روٹی توے سے اتر کر میری پلیٹ میں آجاتی.. یوں میں تین چار روٹیاں کھا جاتا..<p>لیکن مجھے کبھی سمجھ نہ آئی کہ فتح کس کی ھوئی..!!<p>ہمارے گھر کے کچھ اصول تھے.. سب ان پر عمل کرتے تھے.. ھمیں سورج غروب ہونے کے بعد گھر سے باہر نکلنے کی اجازت نہیں تھی.. لیکن تعلیم مکمل کرنے کے بعد مجھے لاھور میں ایک اخبار میں ملازمت ایسی ملی کہ میں رات گئے گھر آتا تھا.. ماں کا مگر وہ ھی معمول رھا..<p>میں فجر کے وقت بھی اگر گھر آیا تو دروازہ خود کھولنے کی ضرورت کبھی نہ پڑی.. لیٹ ہو جانے پر کوشش ھوتی تھی کہ میں خاموشی سے دروازہ کھول لوں تاکہ ماں کی نیند خراب نہ ھو.. لیکن میں ادھر چابی جیب سے نکالتا ‘ ادھر دروازہ کھل جاتا..<p>میں ماں سے پوچھتا تھا.. ” آپ کو کیسے پتہ چل جاتا ھے کہ میں آ گیا ھوں..؟ “<p>وہ ہنس کے کہتی.. ” مجھے تیری خوشبو آ جاتی ھے.. “<p>پھر ایک دن ماں دنیا سے چلی گئی..!!<p>ماں کی وفات کے بعد ایک دفعہ میں گھر لیٹ پہنچا.. بہت دیر تک دروازے کے پاس کھڑا رھا.. پھر ھمت کر کے آہستہ سے دروازہ کھٹکٹایا.. کچھر دیر انتظار کیا اور جواب نہ ملنے پر دوبارہ دستک دی.. پھر گلی میں دروازے کے قریب اینٹوں سے بنی دھلیز پر بیٹھ گیا..<p>سوچوں میں گم نجانے میں کب تک دروازے کے ساتھ ٹیک لگائے بیٹھا رھا اور پتہ نہیں میری آنکھ کب لگی.. بس اتنا یاد ھے کہ مسجد سے آذان سنائی دی.. کچھ دیر بعد سامنے والے گھر سے امّی کی سہیلی نے دروازہ کھولا..<p>وہ تڑپ کر باھر آئیں..<p>انہوں نے میرے سر پر ھاتھ رکھا اور بولیں.. ” پتر ! تیری ماں گلی کی جانب کھلنے والی تمہارے کمرے کی کھڑکی سے گھنٹوں جھانکتی رھتی تھی.. ادھر تو گلی میں قدم رکھتا تھا اور ادھر وہ بھاگم بھاگ نیچے آ جاتی تھیں.. “<p>وہ بولیں.. ” پتر ! تیرے لیٹ آنے ‘ رات گئے کچن میں برتنوں کی آوازوں اور شور کی وجہ سے تیری بھابی بڑی بڑ بڑ کیا کرتی تھی کیونکہ ان کی نیند خراب ہوتی تھی.. “<p>پھر انہوں نے آبدیدہ نظروں سے کھڑکی کی طرف دیکھا اور بولیں..<p>“پتر ! ھن اے کھڑکی کدے نیئں کھلنی..!! “</p>
      <figure>
        <img src="https://i2.wp.com/ehsaas.eu/wp-content/uploads/2017/11/IMG_0699.jpg?resize=1536%2C1013&amp;ssl=1"/>
      </figure>
    </article>
  <div style="clear:both;width:100%;text-align:center; font-size:11px; "><a target="_blank" title="Facebook Auto Publish" href="http://xyzscripts.com/wordpress-plugins/facebook-auto-publish/details" >Facebook Auto Publish</a> Powered By : <a target="_blank" title="PHP Scripts & Programs" href="http://www.xyzscripts.com" >XYZScripts.com</a></div>	<div style="display:none">
	</div>
<div class='code-block code-block-2' style='margin: 8px 0; text-align: right; clear: both;'>
<script async src="//pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>

<ins class="adsbygoogle"
     style="display:block"
     data-ad-client="ca-pub-9565307216854847"
     data-ad-slot="8381744131"
     data-ad-format="auto"></ins>
<script>
(adsbygoogle = window.adsbygoogle || []).push({});
</script></div>
<script type='text/javascript' src='https://ehsaas.eu/wp-includes/js/jquery/jquery.js?ver=1.12.4'></script>
<script type='text/javascript' src='https://ehsaas.eu/wp-includes/js/jquery/jquery-migrate.min.js?ver=1.4.1'></script>
<script type='text/javascript' src='https://ehsaas.eu/wp-content/plugins/jetpack/_inc/build/spin.min.js?ver=1.3'></script>
<script type='text/javascript' src='https://ehsaas.eu/wp-content/plugins/jetpack/_inc/build/jquery.spin.min.js?ver=1.3'></script>
<script type='text/javascript'>
/* <![CDATA[ */
var jetpackCarouselStrings = {"widths":[370,700,1000,1200,1400,2000],"is_logged_in":"","lang":"en","ajaxurl":"https:\/\/ehsaas.eu\/wp-admin\/admin-ajax.php","nonce":"37376cebe7","display_exif":"1","display_geo":"1","single_image_gallery":"1","single_image_gallery_media_file":"","background_color":"black","comment":"Comment","post_comment":"Post Comment","write_comment":"Write a Comment...","loading_comments":"Loading Comments...","download_original":"View full size <span class=\"photo-size\">{0}<span class=\"photo-size-times\">\u00d7<\/span>{1}<\/span>","no_comment_text":"Please be sure to submit some text with your comment.","no_comment_email":"Please provide an email address to comment.","no_comment_author":"Please provide your name to comment.","comment_post_error":"Sorry, but there was an error posting your comment. Please try again later.","comment_approved":"Your comment was approved.","comment_unapproved":"Your comment is in moderation.","camera":"Camera","aperture":"Aperture","shutter_speed":"Shutter Speed","focal_length":"Focal Length","copyright":"Copyright","comment_registration":"0","require_name_email":"1","login_url":"https:\/\/ehsaas.eu\/wp-login.php?redirect_to=https%3A%2F%2Fehsaas.eu%2F2017%2F11%2F30%2F%25d9%25be%25d8%25aa%25d8%25b1-%25da%25be%25d9%2586-%25d8%25a7%25db%2592-%25da%25a9%25da%25be%25da%2591%25da%25a9%25db%258c-%25da%25a9%25d8%25af%25db%2592-%25d9%2586%25db%258c%25d8%25a6%25da%25ba-%25da%25a9%25da%25be%25d9%2584%25d9%2586%25db%258c%2F","blog_id":"1","meta_data":["camera","aperture","shutter_speed","focal_length","copyright"],"local_comments_commenting_as":"<fieldset><label for=\"email\">Email (Required)<\/label> <input type=\"text\" name=\"email\" class=\"jp-carousel-comment-form-field jp-carousel-comment-form-text-field\" id=\"jp-carousel-comment-form-email-field\" \/><\/fieldset><fieldset><label for=\"author\">Name (Required)<\/label> <input type=\"text\" name=\"author\" class=\"jp-carousel-comment-form-field jp-carousel-comment-form-text-field\" id=\"jp-carousel-comment-form-author-field\" \/><\/fieldset><fieldset><label for=\"url\">Website<\/label> <input type=\"text\" name=\"url\" class=\"jp-carousel-comment-form-field jp-carousel-comment-form-text-field\" id=\"jp-carousel-comment-form-url-field\" \/><\/fieldset>"};
/* ]]> */
</script>
<script type='text/javascript' src='https://ehsaas.eu/wp-content/plugins/jetpack/modules/carousel/jetpack-carousel.js?ver=20170209'></script>
</body>
</html>

its arabic text and mostly articles get same error ( General Errors: htmlParseEntityRef: no name )

**Comment by sultansaga** [comment]
No rules defined for <div class="at-above-post addthis_tool"> in the context of InstantArticle
No rules defined for <div class="at-below-post addthis_tool"> in the context of InstantArticle
No rules defined for <div class="jp-relatedposts"> in the context of InstantArticle

**Comment by diegoquinteiro** [comment]
Hi @sultansaga! Thanks for reporting. The issue here is that you use 2 plugins (JP Related Posts and Add This) that our plugin do not understand. A possible fix is going to the Instant Articles plugin settings, and add the following Custom Transformer Rules:

```json
{
    "rules":[
        { "class": "IgnoreRule", "selector": "addthis_tool" },
        { "class": "IgnoreRule", "selector": "jp-relatedposts" }
    ]
}
```

Let me know if that works, otherwise re-open this issue.
